src/test/java/edu/tseidler/model/BoardTest.java

```diff
@@ -94,9 +127,9 @@ public void setBoardWinningNumber(int[] dimensionsAndWinningNumber) {
         board = new Board(dimensionsAndWinningNumber);
 
         SoftAssert sa = new SoftAssert();
```

Comment:
Fajna funkcjonalność NG - pewnie podkradnę :)

Reply:
tak, warto użyć - jak nie przejdzie jakiś podtest, to reszta podtestów się odpali :)
na koniec nie zapomnij uruchomić sa.assertAll(), żeby uruchomić testy!